### PR TITLE
some additions

### PR DESCRIPTION
**Pull request by inamoto4**
I keep working on the person file.

### CODE REVIEW

File: data/Person.csv [hunk 1]
```diff
@@ -180,7 +178,7 @@ P0088,巴巴耶夫斯基,谢苗,zh,male,NULL,NULL,NULL,NULL,1909,2000,,,PL0034,,
 P0088,Бабаевский,Семён,ru,male,NULL,NULL,NULL,NULL,1909,2000,,,PL0034,,,https://en.wikipedia.org/wiki/Semyon_Babayevsky,,,,,,2017-07-10,OS,2018-01-08,MG
 P0089,Bach,Johann Sebastian,en,male,NULL,NULL,NULL,NULL,1685,1750,,,PL0035,,,https://en.wikipedia.org/wiki/Johann_Sebastian_Bach,,,,,,2017-07-10,OS,2018-01-08,MG
 P0089,巴赫,约翰·塞巴斯蒂安,zh,male,NULL,NULL,NULL,NULL,1685,1750,,,PL0035,,,https://en.wikipedia.org/wiki/Johann_Sebastian_Bach,,,,,,2017-07-10,OS,2018-01-08,MG
-P0090,Bai,Juyi,en,male,NULL,NULL,NULL,NULL,0772,0846,,,PL0046,,,https://en.wikipedia.org/wiki/Bai_Juyi,,,,,,2017-07-10,OS,2018-01-10,DP
+P0090,Bai,Juyi,en,male,NULL,NULL,NULL,NULL,772,846,,,PL0046,,,https://en.wikipedia.org/wiki/Bai_Juyi,,,,,,2017-07-10,OS,2018-01-10,DP
```
Comment (duncdrum) [inline data/Person.csv]: this seems an accidental overwrite `yyyy` is the correct date format

File: data/Act.csv [hunk 2]
```diff
@@ -121,7 +121,7 @@ A00119,P0038,,,,,,,T00026,ONLINE SOURCE,,2017-12-17,MG,2017-12-17,MG
 A00120,P0040,,,,,,,T00119,ONLINE SOURCE,,2017-12-17,MG,2017-12-17,MG
 A00121,P0041,,,,,,,T00119,ONLINE SOURCE,,2017-12-17,MG,2017-12-17,MG
 A00122,P0042,,,,,,,T00119,ONLINE SOURCE,,2017-12-17,MG,2017-12-17,MG
-A00123,P0044,,,,,,,T00026,ONLINE SOURCE,,2017-12-17,MG,2017-12-17,MG
+A00123,P0119,,,,,,,T00026,ONLINE SOURCE,,2017-12-17,MG,2017-12-17,MG
```
Comment (duncdrum) [inline data/Act.csv]: this looks like another accident. If it wasn't, please own the change by setting the last_modieied field

File: data/Person.csv (hunk 1 above)
Comment (inamoto4) [reply data/Person.csv]: The thing is that I neither changed the entry of Landport nor that of Bai Juyi. Don't know why this is happening.

File: data/Act.csv (hunk 2 above)
Comment (inamoto4) [reply data/Act.csv]: I forgot to change the last_modified field, thx for reminding.

File: data/Person.csv (hunk 1 above)
Comment (duncdrum) [reply data/Person.csv]: Check your libre office settings and disable all auto substitutions or auto-formatting. Clear all formatting on the tables that your work on before saving just to be sure.

**Pull request merged**